rds/main.tf

```diff
@@ -86,23 +86,24 @@ resource "aws_db_parameter_group" "rds_postgres" {
 }
 
 resource "aws_db_instance" "rds" {
-  identifier              = "${var.project}-${var.environment}${var.tag}-rds${var.number}"
-  allocated_storage       = "${var.storage}"
-  engine                  = "${lookup(var.engines, var.rds_type)}"
-  engine_version          = "${lookup(var.engine_versions, var.rds_type)}"
-  instance_class          = "${var.size}"
-  storage_type            = "${var.storage_type}"
-  username                = "root"
-  password                = "${var.rds_password}"
-  vpc_security_group_ids  = ["${aws_security_group.sg_rds.id}"]
-  db_subnet_group_name    = "${aws_db_subnet_group.rds.id}"
-  parameter_group_name    = "${var.rds_parameter_group_name}"
-  multi_az                = "${var.multi_az}"
-  replicate_source_db     = "${var.replicate_source_db}"
-  backup_retention_period = "${var.backup_retention_period}"
-  storage_encrypted       = "${var.storage_encrypted}"
-  apply_immediately       = "${var.apply_immediately}"
-  skip_final_snapshot     = false
+  identifier                = "${var.project}-${var.environment}${var.tag}-rds${var.number}"
+  allocated_storage         = "${var.storage}"
+  engine                    = "${lookup(var.engines, var.rds_type)}"
+  engine_version            = "${lookup(var.engine_versions, var.rds_type)}"
+  instance_class            = "${var.size}"
+  storage_type              = "${var.storage_type}"
+  username                  = "root"
+  password                  = "${var.rds_password}"
+  vpc_security_group_ids    = ["${aws_security_group.sg_rds.id}"]
+  db_subnet_group_name      = "${aws_db_subnet_group.rds.id}"
+  parameter_group_name      = "${var.rds_parameter_group_name}"
+  multi_az                  = "${var.multi_az}"
+  replicate_source_db       = "${var.replicate_source_db}"
+  backup_retention_period   = "${var.backup_retention_period}"
+  storage_encrypted         = "${var.storage_encrypted}"
+  apply_immediately         = "${var.apply_immediately}"
+  skip_final_snapshot       = "${var.skip_final_snapshot}"
+  final_snapshot_identifier = "${var.project}-${var.environment}${var.tag}-rds${var.number}-final-${md5(timestamp())}"
```

Comment:
```
timestamp() - Returns a UTC timestamp string in RFC 3339 format. This string will change with every invocation of the function, so in order to prevent diffs on every plan & apply, it must be used with the ignore_changes lifecycle attribute.
```